docs/settings.md

```diff
@@ -55,6 +56,11 @@ Setting the TOKEN_TTL to `None` will create tokens that never expire.
 Warning: setting a 0 or negative timedelta will create tokens that instantly expire,
 the system will not prevent you setting this.
 
+## TOKEN_TTL_VISIBILITY
```

Comment:
I'm not 100% sure about the name, as usual naming things is hard...
I feel that we should add a clue that this is related to the login view only.

`LOGIN_TOKEN_TTL_VISIBILITY`
`LOGIN_SHOW_TOKEN_TTL`
`SHOW_TOKEN_TTL_AT_LOGIN` ...

Reply:
`ENABLE_TOKEN_EXPIRY`? Yeah naming is hard. Lets think about it some, maybe @belugame have some input? :-)

Reply:
Mmmm not a  fan of `ENABLE_TOKEN_EXPIRY` it sounds like it would enable / disable the token expiry feature like "a token would never expire when falsy".

`LOGIN_TOKEN_TTL` ?

Also the more I think about the name, the more I'm wondering if we need a setting at all.
This is not backward incompatible as it will just add a key to the payload, not change any of the existing keys + there is no extra performance cost when returning it.

Maybe we should just drop the setting?

Reply:
I do agree that keeping consistency in default values and behaviours is important.

On the other hand, I think we can also safely say that knowing when and if a token expires is not solely linked to the auto refresh feature:

- The token expiry can be used when auto refresh is off, eg: to warn a user that he will be disconnected in x minutes.

- You may also want to let your user know that he needs to explicitly logout when his token never expires.

To make a good argument in favor of a configurable token expiry visibility we would need to find a case against exposing the token expiry and I struggle finding one, any thoughts?